gapic-generator/lib/gapic/presenters/method_rest_presenter.rb

```diff
@@ -91,18 +94,33 @@ def return_type
       #
       def doc_response_type
         return "::Gapic::Operation" if lro?
-        return "::Gapic::Rest::PagedEnumerable<#{pagination.paged_element_doc_type}>" if paged?
+        if paged?
+          elem_type = compute_pagination&.paged_element_doc_type
+          elem_type ||= (lro? ? "::Gapic::Operation" : @main_method.paged_response_type)
```

Comment:
This will always return false

Reply:
Are you sure? It's not possible to be both `paged?` and `lro?`? The logic in `MethodPresenter#lro?` seems to account for the case of both.

Reply:
you're checking for `lro?` 2 lines up and bailing out if it's true though

Reply:
Oh 🤦🏻‍♂️ you're right. Okay I don't understand fully the cases here, but I guess we'll retain the current logic.